dist/common/scripts/scylla_setup

```diff
@@ -62,8 +62,11 @@ def print_non_interactive_suggestion_message(args):
         print('\nAlso, to avoid the time-consuming I/O tuning you can add --no-io-setup and copy the contents of /etc/scylla.d/io*')
         print('Only do that if you are moving the files into machines with the exact same hardware')
 
+def filter_virt_netdev(netdev):
```

Comment:
```suggestion
def is_non_virtual_netdev(netdev):
```

Reply:
Renamed filter_virt_netdev to is_non_virtual_netdev.

Reply:
Reconsindered function name, changed it to is_hw_nic_or_bonding() because it's bit misleading.
What we are checking is, netdev == hw_nic or netdev == bonding.

see https://github.com/scylladb/scylladb/pull/15958#discussion_r1390681211